RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -18,6 +18,8 @@ @implementation RNDeviceInfo
 
 @synthesize isEmulator;
 
+@synthesize isEmulator;
```

Comment:
@GantMan this duplicate synthesize line is breaking the current release

Reply:
EEK!  PR?

Reply:
if not I can knock it out

Reply:
@GantMan sorry about that, didn't have a chance to submit the PR, thanks for jumping on that so fast!